### PR TITLE
Rrogahn/overveiw page

### PR DESCRIPTION
**Pull request by RickRogahn**
Updating the introduction page / overview page.

### CODE REVIEW

File: academic-services/knowledge-exploration-service/introduction.md [hunk 1]
```diff
@@ -1,16 +1,88 @@
...
+
+- *Create knowledge applications* - The set of MAKES APIs that are provisioned allow users to build dynamic knowledge applications that can analyze the state of research in particular areas, compare individuals and institutions and find objective connections between entities.
+
+- *Add a large corpus of academic entities to an existing search solution* - MAKES APIs can be federated to enhance existing search solutions allowing users access to a large corpus of academic research.
+
+- *Create a custom index from a subset of the Microsoft Academic Graph* - In some cases you may not need all of the data in MAG, or you would like combine your own private data with data from MAG.  MAKES allows you to generate an index MAG and private data.  An example would be to build an index specific to certain Journals or Institutions and include publications from your own organization.
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/introduction.md]: "MAKES allows you to generate an index MAG and private data. " => "The MAKES build tool allows users to easily create their own custom indexes, using subsets or super-sets of the Microsoft Academic Graph."

File: academic-services/knowledge-exploration-service/introduction.md [hunk 2]
```diff
@@ -1,16 +1,88 @@
...
+
+## What is MAKES?
+
+The Microsoft Academic knowledge Exploration Service (MAKES) in its' basic form is a self hosted REST API leveraging an index of all the entities in the Microsoft Academic Graph (MAG).  MAKES subscriptions deliver all the components needed to self host instances of this REST API and index.  By subscribing to MAKES, these components are delivered to your Azure subscription when new versions of MAG are created; typically once every 1 to 2 weeks.  Users of MAKES run a provided script and instances are automatically provisioned to their Azure account.
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/introduction.md]: Remove "Users of MAKES run a provided script and instances are automatically provisioned to their Azure account."; doesn't really add any new information

File: academic-services/knowledge-exploration-service/introduction.md [hunk 3]
```diff
@@ -1,16 +1,88 @@
...
+
+The components shipped to a MAKES subscription include:
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/introduction.md]: "The components shipped to a MAKES subscription include:" => "The components include:"

File: academic-services/knowledge-exploration-service/introduction.md [hunk 4]
```diff
@@ -1,16 +1,88 @@
...
+
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/introduction.md]: The "MAKES" prefix for the items below is redundant. I'd suggest removing it as it's not telling the user anything useful

File: academic-services/knowledge-exploration-service/introduction.md [hunk 5]
```diff
@@ -1,16 +1,88 @@
...
+- *MAKES Web Host Application* - This is the REST API web application that interacts with the MAKES index and grammar.
+
+- *MAKES VM Image* - A virtual machine image that is configured to support the MAKES web host application.
+
+- *MAKES Indexes* - Binary index files built from MAG that support the [Knowledge Exploration Service (KES)](#knowledge-exploration-service).
+
+- *MAKES Grammar* - A binary grammar file built to support academic related queries using [KES](#knowledge-exploration-service).
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/introduction.md]: "MAKES Grammar" => "Language model"

Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/introduction.md]: "A binary grammar file built to support academic related queries using" => "MAKES includes a [SRGS](https://en.wikipedia.org/wiki/Speech_Recognition_Grammar_Specification) based language model to support the natural language query processing provided by its Interpret method."

File: academic-services/knowledge-exploration-service/introduction.md [hunk 6]
```diff
@@ -1,16 +1,88 @@
...
+
+## What is MAKES?
+
+The Microsoft Academic knowledge Exploration Service (MAKES) in its' basic form is a self hosted REST API leveraging an index of all the entities in the Microsoft Academic Graph (MAG).  MAKES subscriptions deliver all the components needed to self host instances of this REST API and index.  By subscribing to MAKES, these components are delivered to your Azure subscription when new versions of MAG are created; typically once every 1 to 2 weeks.  Users of MAKES run a provided script and instances are automatically provisioned to their Azure account.
+
+The components shipped to a MAKES subscription include:
+
+- *MAKES Web Host Application* - This is the REST API web application that interacts with the MAKES index and grammar.
+
+- *MAKES VM Image* - A virtual machine image that is configured to support the MAKES web host application.
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/introduction.md]: This is somewhat misleading. MAKES doesn't actually ship with a VM, but instead *generates* it the first time it runs (hence why the first run takes an extra 45 minutes or so)

File: academic-services/knowledge-exploration-service/introduction.md [hunk 7]
```diff
@@ -1,16 +1,88 @@
...
+
+- *MAKES Indexes* - Binary index files built from MAG that support the [Knowledge Exploration Service (KES)](#knowledge-exploration-service).
+
+- *MAKES Grammar* - A binary grammar file built to support academic related queries using [KES](#knowledge-exploration-service).
+
+- *Tools* - Scripts required to automatically provision MAKES to Azure.
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/introduction.md]: "Tools" => "Management tools"

Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/introduction.md]: "Scripts required to automatically provision MAKES to Azure." => "Includes tools that help users customize MAKES and deploy instances to Azure"

File: academic-services/knowledge-exploration-service/introduction.md [hunk 8]
```diff
@@ -1,16 +1,88 @@
...
+
+## MAKES Architecture
+
+The MAKES API is designed to be hosted in an Azure Virtual Machine Scale Set to provide extensibility, scalability, and reliability.  Azure supports virtual machine scale sets with various features out of the box:
+
+- Auto scaling
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/introduction.md]: We should be clear that auto-scaling is not setup by default for users

File: academic-services/knowledge-exploration-service/introduction.md [hunk 9]
```diff
@@ -1,16 +1,88 @@
...
+
+- *MAKES Indexes* - Binary index files built from MAG that support the [Knowledge Exploration Service (KES)](#knowledge-exploration-service).
+
+- *MAKES Grammar* - A binary grammar file built to support academic related queries using [KES](#knowledge-exploration-service).
+
+- *Tools* - Scripts required to automatically provision MAKES to Azure.
+
+## MAKES Architecture
+
+The MAKES API is designed to be hosted in an Azure Virtual Machine Scale Set to provide extensibility, scalability, and reliability.  Azure supports virtual machine scale sets with various features out of the box:
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/introduction.md]: "Azure Virtual Machine Scale Set" => "[Azure Virtual Machine Scale Set](https://docs.microsoft.com/en-us/azure/virtual-machine-scale-sets/overview)"

Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/introduction.md]: "Azure supports virtual machine scale sets with various features out of the box" => "Virtual machine scale sets allow users to easily configure"

File: academic-services/knowledge-exploration-service/introduction.md [hunk 10]
```diff
@@ -1,16 +1,88 @@
...
+
+## Using MAG and MAKES to generate a custom solution
+
+Some customers may have requirements that go beyond hosting the supplied index.  Specific customizations of the MAKES indexes are supported and give the user a great deal of flexibility.  Below is a diagram showing the end to end process users would follow when combining MAG and MAKES subscriptions to create custom solutions with private data and specific entities from MAG:
+
+![Creating Custom solutions with MAG and MAKES](media/combining-mag-and-makes.png "Creating Custom solutions with MAG and MAKES")
+
+This scenario can be taken further as well.  Users are able to modify the grammar and schema of MAKES to support additional languages and data attributes.  These advanced topics are covered in the "how to" section.
+
+## Knowledge Exploration Service
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/introduction.md]: Reading this page over a couple times I think we should just drop this section entirely.

I think it's best to explain it in the context of "advanced" topics, i.e. when we're showing them how to modify schema and/or grammar.

Comment (RickRogahn) [reply academic-services/knowledge-exploration-service/introduction.md]: I agree, can we move the section once we have the advanced topics up?  Currently I'm pointing to this bookmark in a couple different places and can update them all once we find a home.

File: academic-services/knowledge-exploration-service/introduction.md [hunk 11]
```diff
@@ -2,15 +2,85 @@
...
 
-In addition to the academic index included with each release, users can also add their own data to customize the search experience.
+Once signed up for MAKES, new versions of the service and indexes will be delivered to you every 1-2 weeks.  Users have the option of using the default index supplied by the service, which includes all the entities and meta data included in MAG, or using a sub-graph of MAG to create their own custom indexes to power MAKES.  Users are also able to combine private data with the data from MAG to create custom indexes and search experiences for MAKES.
 
-## About Knowledge Exploration Service
+## Why use MAKES?
+
+Here are some key features of the Microsoft Academic Knowledge Exploration Service (MAKES):
+
+- *Self-hosting Project Academic Knowledge* - Many organizations have found great value in our [Academic Knowledge API](https://www.microsoft.com/research/project/academic-knowledge/) (formerly known as the Cognitive Service Academic Knowledge API), but have been limited by the throughput allowed.  MAKES is a turn-key solution to self-host this set of APIs without limits.
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/introduction.md]: >Academic Knowledge API [](start = 96, length = 22)

Project Academic Knowledge API

**Pull request merged**